keyboards/ergodox_ez/keymaps/hacker_dvorak/config.h

```diff
@@ -0,0 +1,76 @@
+/*
+  Set any config.h overrides for your specific keymap here.
+  See config.h options at https://docs.qmk.fm/#/config_options?id=the-configh-file
+*/
+#ifndef CONFIG_HACKER_DVORAK_H
+#define CONFIG_HACKER_DVORAK_H
+
+#include "../../config.h"
+
+// Keyboard Settings
+#undef TAPPING_TERM
+#define TAPPING_TERM 1000
+
+#undef DEBOUNCE
+#define DEBOUNCE 10
+
+#undef ONESHOT_TAP_TOGGLE
+#define ONESHOT_TAP_TOGGLE 1
+
+#undef ONESHOT_TIMEOUT
+#define ONESHOT_TIMEOUT 1000
+
+#undef PERMISSIVE_HOLD
+#define PERMISSIVE_HOLD
+
+#undef IGNORE_MOD_TAP_INTERRUPT
+#define IGNORE_MOD_TAP_INTERRUPT
+
+#undef NO_DEBUG
+#define NO_DEBUG
+
+#undef NO_PRINT
```

Comment:
This isn't defined by if `CONSOLE_ENABLE` isn't set to "yes", so you don't need this, at all.
```suggestion
```